.deps/problems.md

```diff
@@ -2,18 +2,20 @@
 
 ## UNRESOLVED Production dependencies
 
-1. `@patternfly/react-table@4.29.0`
-2. `immutable@4.0.0-rc.12`
+1. `@eclipse-che/devworkspace-client@0.0.1-1625495079`
```

Comment:
should be fixed in https://github.com/eclipse-che/che-dashboard/blame/main/.deps/EXCLUDED/prod.md#L6

Reply:
this isn't a critical issue so let's provide the fix in one of the following PRs